src/functional_test_runner/lib/mocha/reporter/reporter.js

```diff
@@ -80,7 +81,11 @@ export function MochaReporterProvider({ getService }) {
                   ? this.runner.test.parent
                   : this.runner.suite;
 
-                recordLog(currentSuite, chunk);
+                // We are registering the current time in order to be able
+                // to extend the log chunk with it so we can have the test
+                // results log lines labeled with such info
+                const currentTime = `${moment().format('HH:mm:ss')}`;
+                recordLog(currentSuite, `${currentTime} ${chunk}`);
```

Comment:
Can we please split up the `chunk` into lines and prefix each line with the current time?

Reply:
Unless the chunk is always a line, in which case can we could rename the variable to `line`

Reply:
@spalger I just realised that the thing we are calling `chunk` is always a line. The current result is like: https://kibana-ci.elastic.co/job/elastic+kibana+pull-request+multijob-intake/6111/testReport/API%20Integration%20Tests/test_api_integration_apis_elasticsearch_index%C2%B7js/apis_elasticsearch_allows_msearch/

I will change `chunk` -> `line`

Good point on that 👍 😃